CHANGES.md

```diff
@@ -1,3 +1,27 @@
+## edge-19.3.2
+
+* Controller
+  * **Breaking change** Removed support for running the control plane in
```

Comment:
I would like to add a little explanation here, like "This simplifies things considerably, and instead, you can install Linkerd in a parallel namespace" or something like that?

Reply:
That's sort of the idea. The main benefit here is that single-namespace mode was feature poor to the point that it wasn't useful, and there was a large overhead to maintaining it, so we removed it. That said, if a user required single-namespace mode, we don't have another alternative to offer them at this point. Will see if I can wordsmith something like that.

Reply:
How about?

```
  * **Breaking change** Removed support for running the control plane in
    single-namespace mode, which was severely limited in the number of features
    it supported due to not having access to cluster-wide resources
```

Reply:
LGTM